CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 Kafka features are not tied to a specific kafka-docker version (ideally all changes will be merged into all branches). Therefore, this changelog will track changes to the image by date.
 
+TBC
+---
+
+-	Issue #208 - Add `KAFKA_CREATE_TOPICS_SEPARATOR` to allow custom input, such as multiline YAML.
```

Comment:
nit: spaces between `-` and "Issue" + s/multiline/multi-line

Reply:
> nit: spaces between - and "Issue"

It's a tab - and it's [markdownfmt](https://github.com/shurcooL/markdownfmt) auto-formatted and consistent with the rest of the file.

I'll fix the "multi-line"